relax/utils.py

```diff
@@ -138,3 +139,39 @@ def default(cls) -> Config:
 # %% ../nbs/00_utils.ipynb 36
 def get_config() -> Config: 
     return main_config
+
+# %% ../nbs/00_utils.ipynb 37
+def set_config(
```

Comment:
Write documentation for this function. See [jax-relax - Contribute (birkhoffg.github.io)](https://birkhoffg.github.io/jax-relax/tutorials/contribution.html#write-documentations-in-jupyter-notebook)

So change to:

```python
def set_config(
    *,
    rng_reserve_size: int = None, # Set the number of random number generators to reserve.
    global_seed: int = None, # Set the global seed for random number generators.
    **kwargs
) -> None:
    """Sets the global configurations."""
```